lib/beaker/options/parser.rb

```diff
@@ -228,10 +228,23 @@ def parse_hosts_options
         hosts_options = Beaker::Options::HostsFileParser.parse_hosts_file(@options[:hosts_file])
         return hosts_options
       rescue Errno::ENOENT
+        error_message = "\nHosts file '#{@options[:hosts_file]}' does not exist."
+        error_message << "\nTrying as beaker-hostgenerator input.\n\n"
+        $stderr.puts error_message
         require 'tempfile'
```

Comment:
I know you didn't change this line in this PR, but is this require still necessary if we're not writing to disk anymore?